elastic/logs/track.json

```diff
@@ -7,9 +7,13 @@
 {% set p_num_query_workflows = p_query_workflows | length %}
 {% set num_corpus = 18 %}
 {% set p_bulk_indexing_clients = (bulk_indexing_clients | default(8)) %}
+{% set p_search_query_clients = (search_query_clients | default(1)) %}
```

Comment:
What do you think about naming the parameter `search_clients` to be consistent with the [API](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/examples.html#ex-search)?